examples/network_transport/transport_dtls_mbedtls.c

```diff
@@ -733,43 +731,37 @@ int32_t DTLS_VerifyRemoteCertificateFingerprint( DtlsSSLContext_t * pSslContext,
         LogError( ( "invalid input, pSslContext || pExpectedFingerprint == NULL || CERTIFICATE_FINGERPRINT_LENGTH < fingerprintMaxLen(%u)", fingerprintMaxLen ) );
         retStatus = -1;
     }
-    else
```

Comment:
Is the else not needed by coverity?

Reply:
That's only needed when there is at least one `else if` there.